src/js/components/Widgets/EndorsementCard.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@material-ui/core';
+import OpenExternalWebSite from '../../utils/OpenExternalWebSite';
+
+class EndorsementCard extends Component {
```

Comment:
I've been doing some thinking about this, and I think it would be better to refactor this EndorsementCard component to use `import { Button } from 'react-bootstrap';` and use the same approach that you do in Widgets/AddEndorsements.jsx

This will allow us to maintain the same look and feel for both components.

Reply:
Also, we have standardized on this approach to taking in proptypes, inside the component:
```
  static propTypes = {
    buttonText: PropTypes.string,
    title: PropTypes.string,
    bsPrefix: PropTypes.string,
    text: PropTypes.string,
  };
```